### PR TITLE
feat: PPANTT-115 Introduced payment-options infra

### PR DESCRIPTION
**Pull request by alessio-cialini**
### List of changes

- Introducing payment-options infra domain

### TODO
`terraform apply`:
- [x] DEV
- [x] UAT
- [ ] PROD

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/paymentoptions-secrets/99_locals.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+locals {
+  project = "${var.prefix}-${var.env_short}-${var.location_short}-${var.domain}"
+  product = "${var.prefix}-${var.env_short}"
+
+  monitor_action_group_slack_name = "SlackPagoPA"
+  monitor_action_group_email_name = "PagoPA"
```
Comment (jacopocarlini) [inline src/domains/paymentoptions-secrets/99_locals.tf:6]:
```suggestion
```
remove these lines. They aren't used.

File: src/domains/paymentoptions-common/10_github_identity.tf [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+  resource_group_name = "${local.product}-${var.location_short}-${var.domain}-sec-rg"
+}
+
+# repos must be lower than 20 items
+locals {
+  repos_01 = [
+    "pagopa-print-payment-notice-service",
+    "pagopa-print-payment-notice-generator",
+    "pagopa-print-payment-notice-functions",
+    "pagopa-template-notice-pdf"
```
Comment (jacopocarlini) [inline src/domains/paymentoptions-common/10_github_identity.tf:21]: please put the right repo names

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
@alessio-cialini can you apply in DEV?

PS: see the comments

File: src/domains/paymentoptions-app/04_apim_notice_generator.tf
Comment (jacopocarlini) [inline src/domains/paymentoptions-app/04_apim_notice_generator.tf:1]: rename this file please

File: src/domains/paymentoptions-app/04_apim_notice_generator.tf [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    display_name = "Payment Options Product pagoPA"
+    description  = "API for Payment Options"
+  }
+}
+
+module "apim_payment_options_product" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_product?ref=v8.18.0"
+  count  = var.is_feature_enabled.paymentoptions ? 1 : 0
+
+  product_id   = "pagopa_notices_generator"
```
Comment (jacopocarlini) [inline src/domains/paymentoptions-app/04_apim_notice_generator.tf:12]: 👀 change this id

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

**Review by jacopocarlini — APPROVED** [review]

**Review by pasqualespica — APPROVED** [review]

**Pull request merged**